doc/tutorial.rst

```diff
@@ -644,18 +651,20 @@ code?
 Kaleidoscope interpreter
 ------------------------
 
-At the moment, the generated library uses the Ada programming language and its
-API isn't stable yet. However, it also exposes a C API and a Python one on the
-top of it. Let's use the Python API for now as it's more concise, handier and
-likely more stable. Besides, using the Python API makes it really easy to
-experiment since you have an interactive interpreter. So, considering you
-successfully built the library with the Kaleidoscope parser and lexer, make
-sure the ``build/lib/langkit_support.relocatable`` and the
+At the moment, the generated library uses the Ada programming language
+and its API isn't stable yet. However, it also exposes a C API and a
+Python one on the top of it. Let's use the Python API for now as it's
+more concise, handier and likely more stable. Besides, using the
+Python API makes it really easy to experiment since you have an
+interactive interpreter. So, considering you successfully built the
+library with the Kaleidoscope parser and lexer, make sure the
+``build/lib/relocatable/dev/libkaleidoscopelang`` and the
 ``build/lib/libkaleidoscopelang.relocatable`` directories is in your
-``LD_LIBRARY_PATH`` (on most Unix, ``DYLD_FALLBACK_LIBRARY_PATH`` on Darwin,
-adapt for Windows) and that the ``build/python/libkaleidoscopelang.py`` is
-reachable from Python (add ``build/python`` in your ``PYTHONPATH`` environment
-variable).
+``LD_LIBRARY_PATH`` (on most Unix, ``DYLD_FALLBACK_LIBRARY_PATH`` on
+Darwin, adapt for Windows) and that the
+``build/python/libkaleidoscopelang/__init__.py`` is reachable from
+Python (add ``build/python/libkaleidoscopelang`` in your
```

Comment:
I'm surprised that one should add `build/python/libkaleidoscopelang` to PYTHONPATH instead of `build/python`, could you double check that please?

Reply:
double checked:
```
$ ls build/python/
libkaleidoscopelang  setup.py
```

Reply:
my bad, you're right!